src/nfa/operations.cc

```diff
@@ -34,9 +37,14 @@ using namespace Mata::Nfa;
 using Mata::Symbol;
 using Mata::BoolVector;
 
+// Vector of bool is probably slower than vector of char (from some little experiment).
+// Just BoolArray would be better?
+// It would make sense to make this type name global, it is type that we use globally to represent bool vectors. ?
 using StateBoolArray = std::vector<bool>; ///< Bool array for states in the automaton.
 
 namespace {
+    //Remove direct_, there are no close alternatives for NFA (unlike for Buchi). Remove "compute_" ?
+    //Is this declared in a header file? Can't find.
```

Comment:
That is because this is in an anonymous namespace. This function is not visible from outside of this very source file. It is not in the interface of the library.

Reply:
Then, this is the same discussion as in one other conversation here about the prefixes for function names. I will close this discussion here, and we will continue the discussion in the other conversation.